apps/explorer/src/app/components/markets/market-details.tsx

```diff
@@ -0,0 +1,246 @@
+import {
+  addDecimalsFormatNumber,
+  formatNumberPercentage,
+  getMarketExpiryDateFormatted,
+  t,
+} from '@vegaprotocol/react-helpers';
+import type { MarketInfoQuery } from '@vegaprotocol/market-info';
+import { MarketInfoTable } from '@vegaprotocol/market-info';
+import pick from 'lodash/pick';
+import {
+  MarketStateMapping,
+  MarketTradingModeMapping,
+} from '@vegaprotocol/types';
+import { AssetDetailsTable, useAssetDataProvider } from '@vegaprotocol/assets';
+import { Splash } from '@vegaprotocol/ui-toolkit';
+import BigNumber from 'bignumber.js';
+import { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+
+export const MarketDetails = ({
```

Comment:
**suggestion (non-blocking)**: This component should have a max width on it. 

There's no current standard - lots of other pages need it to, so this could be a follow up ticket. But something like `xl:max-w-3xl`.

<img width="1483" alt="Screenshot 2023-02-14 at 18 13 30" src="https://user-images.githubusercontent.com/6678/218823637-56d47a4e-0b8b-4619-8796-f3b13b26e999.png">

(Note: I'm not pretending to have a giant monitor, but I zoomed out to emphasies the problem)

Reply:
Created a ticket https://github.com/vegaprotocol/frontend-monorepo/issues/2916

Reply:
Yeah, agreed